config/default.yml

```
@@ -362,6 +362,11 @@ Rails/IgnoredSkipActionFilterOption:
   Include:
     - app/controllers/**/*.rb
 
+Rails/ImplementedMigration:
+  Description: 'Checks for migrations that have a typo in a method name or are half implemented.'
```

Can you update the description?

```suggestion
  Description: 'Checks whether the migration implements either a `change` method or both an `up` and a `down` method.
```